.travis/install.sh

```diff
@@ -50,3 +50,15 @@ pip install -r requirements.d/development.txt
 pip install codecov
 python setup.py --version
 pip install -e .[fuse]
+
+if [ -d "$HOME/.rustup/toolchains" ] && [ -f "$HOME/.cargo/env" ]; then
+    source ~/.cargo/env
+    rustup toolchain install stable # also updates
+    rustup default stable
+else
+    curl https://sh.rustup.rs -sSf | sh -s -- -y
+    source ~/.cargo/env
+fi
+
+cd test-wrapper
+cargo build --release
```

Comment:
usually it is all in the Vagrantfile, but as long as test-wrapper is not stable, just not having to copy and paste these 4 lines would be already an improvement.

once it is stable, it would go into Vagrantfile.